app/src/main/java/org/mozilla/fenix/compose/SectionHeader.kt

```diff
@@ -31,7 +29,7 @@ fun SectionHeader(
         modifier = modifier,
         color = FirefoxTheme.colors.textPrimary,
         fontSize = 16.sp,
-        fontFamily = FontFamily(Font(R.font.metropolis_semibold)),
+        fontWeight = FontWeight.Medium,
```

Comment:
Curious shouldn't this be `FontWeight.SemiBold` to match?

Reply:
Nevermind, I see the comments for this. I am thinking we should post some screenshots for @cwzilla to just super confirm the changes by showing what the header looks like with `FontWeight.SemiBold` and `FontWeight.Medium`

Reply:
Metropolis Semibold|Medium|Semibold
---|---|---
<img width="441" alt="Screen Shot 2022-02-03 at 12 25 01 PM" src="https://user-images.githubusercontent.com/1190888/152396242-0fd2783d-a1c4-434a-8c97-ba4e7ee143c8.png">|<img width="441" alt="Screen Shot 2022-02-03 at 12 23 50 PM" src="https://user-images.githubusercontent.com/1190888/152396184-17f6f2ea-027d-40f6-b6d0-c70d112b3c41.png">|<img width="441" alt="Screen Shot 2022-02-03 at 12 24 38 PM" src="https://user-images.githubusercontent.com/1190888/152396205-80c069c2-7c80-4711-92e2-db16ae05a032.png">

Reply:
I just realized this was a misunderstanding. The screenshots are of the recently visited header, not of a `SectionHeader`.

Reply:
Oh derp, you're right. What I have screenshot is Header16TextStyle in `styles.xml`. I think issue still lies on the fact that the "Jump back in", "Recently visited" headers don't quite match in the 'look' of the original Metropolis Semibold, and there lies the problem with the mismatch on what we are seeing and what was expected when you look at Roboto with 500 font weight in https://fonts.google.com/specimen/Roboto.

Reply:
In this case, it doesn't appear as if "Jump back in" is even bolded. I am testing this on a Pixel 4.

Reply:
Yes, I am currently working on a workaround. I will post the results below.